internal/telemetry/event_test.go

```diff
@@ -0,0 +1,74 @@
+package telemetry
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/google/go-cmp/cmp"
```

Comment:
add a newline above this import to separate out project imports and 3rd party library imports

in general, we have an import pattern across our realm repos:

```go
import (
  // go standard library imports

  // project imports

  // third party library imports
)
```